src/dmd/json.d

```diff
@@ -391,26 +397,26 @@ public:
         }
     }
 
-    void property(const(char)* name, Type type)
+    extern(D) void property(const(char)[] name, Type type)
     {
         if (type)
         {
-            property(name, type.toChars());
+            property(name, RootObject.toString(type));
```

Comment:
Not too thrilled about this. It should be type.toString(). Yes, the C++ code cannot call it, but all you need to do is put a dummy function declaration there on the C++ side so the vtbl[] entries line up.

Reply:
Fixed